spec/site_prism/page_objects/page/result_filters/subject_page.rb

```diff
@@ -13,8 +13,8 @@ class Subject < SitePrism::Section
           end
 
           element :legend, '[data-qa="subject_area__legend"]'
-          element :name, '[data-qa="subject_area__name"]'
-          element :accordion_button, '[data-qa="subject_area__accordion_button"]'
+          element :name, '.govuk-accordion__section-heading'
+          element :accordion_button, '.govuk-accordion__section-button'
```

Comment:
I don’t mind this; I’d be happy of we removed many of the `data-qa` attributes we have on Find as they are usually superfluous.